### PR TITLE
Added parser code for master branch

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Please merge in master

### CODE REVIEW

File: insights/specs/__init__.py [hunk 1]
```diff
@@ -231,6 +231,7 @@ class Specs(SpecSet):
     nfs_exports_d = RegistryPoint(multi_output=True)
     nfs_exports = RegistryPoint()
     nginx_conf = RegistryPoint(multi_output=True)
+    nmcli_dev_show = RegistryPoint()
```
Comment (bfahr) [inline insights/specs/__init__.py]: @vishwanathjadhav you also need to add `nmcli_dev_show` to `insights/specs/defaults.py` as a `simple_command`.

File: insights/parsers/nmcli.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    """
+    This class will parse the output of command ``nmcli dev show``, and the information
+    will be stored in dictionary format.
+
+    NetworkManager displays all the devices and their current states along with network
+    configuration and connection status.
+
+    Attributes:
+        data (dict): Dictionary of keys with values in dict.
+        get_connected_devices(list): list of devices who's state is connected.
```
Comment (bfahr) [inline insights/parsers/nmcli.py]: I believe this should be `connected_devices` instead shouldn't it?

File: insights/specs/__init__.py (hunk 1 above)
Comment (xiangce) [reply insights/specs/__init__.py]: @vishwanathjadhav - and please add its corresponding "simple_file" to `insights/specs/insights_archive.py` and `insights/specs/sos_archive.py` (if necessary)

**Comment by xiangce** [comment]
@bfahr - seems a new robot checker named "default" (build rpm egg by jenkins)  is added, and the check build-check failed, I cannot log in the check page. Anything we should do to correct it?

**Comment by xiangce** [comment]
As per Bob's suggestion, will ignore the `Jenkins build fails` temporary.

**Pull request merged**